src/components/NavBar/ChainSwitcher.tsx

```diff
@@ -79,7 +79,7 @@ export const ChainSwitcher = ({ leftAlign }: ChainSwitcherProps) => {
   const isSupported = !!info
 
   const dropdown = (
-    <NavDropdown top="56" left="0">
+    <NavDropdown top="56" left={leftAlign ? '0' : 'auto'} right={leftAlign ? 'auto' : '0'}>
```

Comment:
this seems unrelated?

Reply:
I noticed a small bug while working on this. Given that it was a two line fix it didn't seem like a valuable use of time to separate into its own PR